api/src/main/java/org/eclipse/microprofile/config/package-info.java

```diff
@@ -1,6 +1,6 @@
 /*
  *******************************************************************************
- * Copyright (c) 2016-2017 Contributors to the Eclipse Foundation
```

Comment:
The package version lower down in this file should be bumped

Reply:
Should the package versions for `org.eclipse.microprofile.config.inject` and `org.eclipse.microprofile.config.spi` also be increased? @Emily-Jiang ?

Reply:
Probably this...
`org.eclipse.microprofile.config` = 1.1
`org.eclipse.microprofile.config.inject` = 1.1
`org.eclipse.microprofile.config.spi` = 1.4